src/Select/Select.component.tsx

```diff
@@ -370,6 +374,20 @@ export class CustomSelect extends React.Component<Props> {
           }
         : {};
 
+    const _components = Object.keys(components).reduce((res, key) => {
+      const Component = { ...components, ...propsComponents }[key];
+      res[key] = ({ children, ...rest }) => {
+        return (
+          <SIdWrapper id={`${id}-${key}`}>
+            <Component {...rest}>{children}</Component>
+          </SIdWrapper>
+        );
+      };
+      return res;
+    }, {});
+
+    console.log(components, propsComponents, _components);
```

Comment:
unwanted logs to console in PR.

Also ran the app and the following features break:
the placeholder Select... is being cut out 
also the props breaks the select with checkbox feature